charts/rapid-response/values.yaml

```diff
@@ -6,6 +6,40 @@ global:
   image:
     pullSecrets: []
     pullPolicy: IfNotPresent
+  ssl:
+    ca:
+      # For outbound connections (secure backend, proxy,...)
+      # A PEM-encoded x509 certificate.  This can also be a bundle with multiple certificates.
+      # See below in the rapidResponse.ssl.ca.cert key for an example
+      cert:
+
+      # Example of a certificate
+      # cert: |
+      #   -----BEGIN CERTIFICATE-----
+      #   MIIDEzCCAfugAwIBAgIQKiv9U+KxPJzu1adXwC06RzANBgkqhkiG9w0BAQsFADAU
+      #   MRIwEAYDVQQDEwloYXJib3ItY2EwHhcNMjIwMjIzMDY1NjExWhcNMjMwMjIzMDY1
+      #   NjExWjAUMRIwEAYDVQQDEwloYXJib3ItY2EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+      #   MMNlTAQ9fvdNOTzZntye0PQYR5SR13E=
+      #   -----END CERTIFICATE-----
+      #   -----BEGIN CERTIFICATE-----
+      #   MIIDEzCCAfugAwIBAgIQKiv9U+KxPJzu1adXwC06RzANBgkqhkiG9w0BAQsFADAU
+      #   MRIwEAYDVQQDEwloYXJib3ItY2EwHhcNMjIwMjIzMDY1NjExWhcNMjMwMjIzMDY1
+      #   NjExWjAUMRIwEAYDVQQDEwloYXJib3ItY2EwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+      #   MMNlTAQ9fvdNOTzZntye0PQYRTTS34D=
+      #   -----END CERTIFICATE-----
+
+      # Filename that is used when creating the secret.  Required if cert is provided.
+      keyName: root_ca_file.crt
```

Comment:
Remove the default here (the default is missing in all the other entries)

Reply:
Removing the default here will cause a BC since the current version of `rapid-response` already accept the `.Values.ssl.ca.certs` and automatically names it in https://github.com/sysdiglabs/charts/pull/961/files#diff-4dd3290c87066c244c1311d77bc14f6a87e30ceb1bdf42ca0f67239c738854e6L38

Reply:
Ok. Semi-related question: why we don't have the default file name in all the charts? 😅

Reply:
Cause almost everywhere else we didn't have this functionality yet...the name has been added here to not cause any issue to who's already using this on `rapid-response`.